cpp/ql/lib/CHANGELOG.md

```diff
@@ -6,7 +6,7 @@
 
 ### Deprecated APIs
 
-* The `NonThrowing` class (`semmle.code.cpp.models.interfaces.NonThrowing`) has been deprecated. Please use the `NonCppThrowingFunction` class instead.
+* The `NonThrowingFunction` class (`semmle.code.cpp.models.interfaces.NonThrowing`) has been deprecated. Please use the `NonCppThrowingFunction` class instead.
```

Comment:
Should the bit in brackets be `semmle.code.cpp.models.interfaces.NonThrowing` or `semmle.code.cpp.models.interfaces.NonThrowing.NonThrowingFunction`?

Reply:
I read it as referring to the file, but I'm fine with whatever you think is best.

Reply:
Slight preference for the full path, but I don't have strong feelings either way.

Reply:
Changed it to `semmle.code.cpp.models.interfaces.NonThrowing.NonThrowingFunction` (note that this dismissed your approval).